tools/sidekick/config.json

```diff
@@ -8,6 +8,16 @@
       "environments": [ "edit" ],
       "url": "/tools/sidekick/library.html",
       "includePaths": [ "**.docx**" ]
+    },
+    {
+      "id": "asset-library",
+      "title": "My Assets",
```

Comment:
AEM Assets library seems more suitable for the title.
Looking for suggestion

Reply:
yes, change it to AEM Assets library